examples/README.md

```diff
@@ -10,39 +10,19 @@ This folder might use unreleased code. View the examples for the latest release
 > There are a few workaround for this problem:
 >
 > - View the examples as they were when the latest version was release by selecting the tag that
->   matches that version. E.g. <https://github.com/ratatui/ratatui/tree/v0.26.1/examples>.
+>   matches that version. E.g. <https://github.com/ratatui-org/ratatui/tree/v0.26.1/examples>.
```

Comment:
I think there's an issue with this PR because it "regresses" a number of changes to the code. 

Here's the simplest example - `ratatui-org` was renamed to `ratatui` and this PR undoes that renaming. I'm seeing a lot of changes that undoes this renaming, changes that undoes typo fixes etc, changes that undoes documentation additions etc. 

It's hard to make out the exact diff associated with this particular change.